extensions/aaudio-cffi.lisp

```diff
@@ -208,20 +208,21 @@
   (:center-rear         #b0000000000000000000000100000000)
   (:left-side           #b0000000000000000000001000000000)
   (:right-side          #b0000000000000000000010000000000)
-  (:center-front-top    #b0000000000000000000100000000000)
+  (:center-top          #b0000000000000000000100000000000)
   (:left-front-top      #b0000000000000000001000000000000)
-  (:right-front-top     #b0000000000000000010000000000000)
-  (:left-rear-top       #b0000000000000000100000000000000)
-  (:center-rear-top     #b0000000000000001000000000000000)
-  (:right-rear-top      #b0000000000000010000000000000000)
-  (:left-side-top       #b0000000000000100000000000000000)
-  (:right-side-top      #b0000000000001000000000000000000)
-  (:left-front-bottom   #b0000000000010000000000000000000)
-  (:center-front-bottom #b0000000000100000000000000000000)
-  (:right-front-bottom  #b0000000001000000000000000000000)
-  (:subwoofer-2         #b0000000010000000000000000000000)
-  (:left-front-wide     #b0000000100000000000000000000000)
-  (:right-front-wide    #b0000001000000000000000000000000))
+  (:center-front-top    #b0000000000000000010000000000000)
+  (:right-front-top     #b0000000000000000100000000000000)
+  (:left-rear-top       #b0000000000000001000000000000000)
+  (:center-rear-top     #b0000000000000010000000000000000)
+  (:right-rear-top      #b0000000000000100000000000000000)
+  (:left-side-top       #b0000000000001000000000000000000)
+  (:right-side-top      #b0000000000010000000000000000000)
+  (:left-front-bottom   #b0000000000100000000000000000000)
+  (:center-front-bottom #b0000000001000000000000000000000)
+  (:right-front-bottom  #b0000000010000000000000000000000)
+  (:subwoofer-2         #b0000000100000000000000000000000)
+  (:left-front-wide     #b0000001000000000000000000000000)
+  (:right-front-wide    #b0000010000000000000000000000000))
```

Comment:
No. I *specifically* wrote them the way I did to make them compatible with the scheme used in libmixed: https://github.com/Shirakumo/cl-mixed/blob/master/low-level.lisp#L207

Reply:
Ok, but there was one missing value
From https://developer.android.com/ndk/reference/group/audio#summary :
>   AAUDIO_CHANNEL_TOP_CENTER = 1 << 11,
>  AAUDIO_CHANNEL_TOP_FRONT_LEFT = 1 << 12,
>  AAUDIO_CHANNEL_TOP_FRONT_CENTER = 1 << 13,
>  AAUDIO_CHANNEL_TOP_FRONT_RIGHT = 1 << 14,

Reply:
Oh, dang. My bad! Again, sorry for the false flag.